introduction.tex

```diff
@@ -55,15 +55,19 @@ \subsection{Scenario Descriptions}
 provides natural uranium, which is enriched by an 'enrichment'
 facility to \gls{UOX}, while disposing enrichment waste (tailings)
 to the 'sink' facility. The enriched \gls{UOX} is used
-in the \gls{LWR}s and \gls{UOX} waste is produced. Then the waste
-would either go directly to the sink (scenario 1), or is reprocessed
-for further use (scenario 2 and 3). In scenario 2, the waste is 
-separated in the separations facility (reprocessing plant), into
-two streams, Plutonium and Uranium. The plutonium is mixed with depleted uranium from enrichment
-to create \gls{MOX}, and the reprocessed uranium is stockpiled in the fabrication facility.
-The produced \gls{MOX} is then used in French \gls{PWR}s.
-In scenario 3, the waste goes through the same process but is mixed
-to create \gls{SFR} fuel.
+in the \gls{LWR}s and \gls{UOX} waste is produced. Then the spent fuel
+goes directly to the sink. However, in the case of France, the spent fuel
+is reprocessed to be fabricated into \gls{MOX}.
+
+
+This paper explores two scenarios.
+First is the case of having regional repositories. Taking into consideration 
```

Comment:
X is a placeholder right? I should eventually figure out how many regional repositories are to be built and replace the X with an actual number?